pages/edge/configuration/app-configuration.mdx

```diff
@@ -44,3 +44,14 @@ The `description` field is used to provide a description of the application. It
 The `package` field is used to provide the primary workload to execute. It is **required** and must be a valid WebAssembly package identifier.
 
 - Required: `true`
+
+## Sample Configuration
+
+This sample configuration is for a python based flask application.
+
+```yaml copy
+---
+kind: wasmer.io/App.v0
+name: wasmer-python-flask-server-myapp-example
+package: wasmer/python-flask-server-myapp-example@0.1.0
```

Comment:
This package is the right one? Why wasmer is the owner?

Reply:
Because the examples live on wasmer repo

Reply:
Plus, This one is just a sample config